sql/log.cc

```diff
@@ -2921,6 +2920,40 @@ bool MYSQL_LOG::init_and_set_log_file_name(const char *log_name,
   return FALSE;
 }
 
+/** Shut down logging and complain part 2 of the original fatal_log_error msg */
+void MYSQL_LOG::exec_error_action()
```

Comment:
I see we have many types of logs.
I wonder how many of them benefits from a config similar to binary logs’ `binlog_error_action`?